packages/ui/client/composables/browser.ts

```diff
@@ -0,0 +1,36 @@
+import { detailSizes } from '~/composables/navigation'
+
+export type ResizingListener = (isResizing: boolean) => void
+
+const resizingSymbol = Symbol.for('resizing')
+
+export function recalculateDetailPanels() {
+  const iframe = document.querySelector('#tester-ui iframe[data-vitest]')!
+  const panel = document.querySelector('#details-splitpanes')!
+  const panelWidth = panel.clientWidth
+  const iframeWidth = iframe.clientWidth
+  const iframePercent = Math.min((iframeWidth / panelWidth) * 100, 95)
+  const detailsPercent = 100 - iframePercent
+  detailSizes.value = [iframePercent, detailsPercent]
+}
+
+export function registerResizingListener(listener: ResizingListener) {
+  inject<(listener: ResizingListener) => void>(resizingSymbol)?.(listener)
```

Comment:
isn't this just a `ResizingListener`?

Reply:
I'm goibg to configure it globally (like navigation conposable), thebuinis just a spa, dont need inject/provide

Reply:
Thisblistener is the callback called from the index.vue

Reply:
I mean the type is repeated, you already defined it on the top